var/spack/repos/builtin/packages/esmf/package.py

```diff
@@ -29,6 +29,9 @@ class Esmf(MakefilePackage):
     # Develop is a special name for spack and is always considered the newest version
     version("develop", branch="develop")
     # generate chksum with 'spack checksum esmf@x.y.z'
+    version("8.7.0b04", commit="609c81179572747407779492c43776e34495d267")
```

Comment:
We don't need 8.7.0b04 for release 1.6.0

Reply:
removed